roxie/ccd/ccdmain.cpp

```diff
@@ -1355,28 +1359,39 @@ int CCD_API roxie_main(int argc, const char *argv[], const char * defaultYaml)
                     if (!roxiePort)
                     {
                         roxiePort = port;
+                        if (roxieFarm.getPropBool("@tls"))
+                            roxiePortTlsClientConfig = createTlsClientSecretInfo(roxieFarm.queryProp("@issuer"), roxieFarm.getPropBool("@pulic")==false, roxieFarm.getPropBool("@selfSigned"));
```

Comment:
typo: "@pulic".
!x.getPropBool() is also more common/clearer than x.getPropBool()==false?